orbit/pkg/table/mdm/mdm_windows.go

```diff
@@ -165,6 +287,39 @@ func getEnrollmentInfo() (uint32, string, error) {
 	return isDeviceRegisteredWithMDM, uriData, nil
 }
 
+// isReadOnlyCommandRequest returns true if the verbs used on input SyncML commads are only Get
+func isReadOnlyCommandRequest(inputCmd string) (bool, error) {
+	if len(inputCmd) == 0 {
+		return false, errors.New("empty input command")
+	}
+
+	// creating a new SyncMLBody message object
+	messageObject := new(SyncMLBody)
+
+	// parsing output SyncML message
+	d := xml.NewDecoder(bytes.NewReader([]byte(inputCmd)))
+	d.CharsetReader = identReader
+
+	// decoding the XML message
+	if err := d.Decode(messageObject); err != nil {
+		return false, err
+	}
+
+	// getting response data from output message
+	if len(messageObject.Item) > 0 {
```

Comment:
Shouldn't we do:
```go
if len(messageObject.Item) == 0 {
  return false, nil
}
```
to be on the safe side.

Reply:
good catch! I've just addressed and tested this one. 

> And moreover looking at the for loop below, if it didn't find a Get item also return false, nil.
Yes, the idea is to return immediately if there is a command verb other than "Get"